llvm/lib/Target/RISCV/RISCVSystemOperands.td

```diff
@@ -158,6 +158,7 @@ def : SysReg<"hip", 0x644>;
 def : SysReg<"hvip", 0x645>;
 def : SysReg<"htinst", 0x64A>;
 def : SysReg<"hgeip", 0xE12>;
+def : SysReg<"hedelegh", 0x612>;
```

Comment:
Need `let isRV32Only = 1 in`

Should this be in the same group as "hedeleg"

Reply:
Still need to move the "Hypervisor Trap Setup" section instead of "Hypervisor Trap Handling". It belongs in the same group as "hedeleg"